support-frontend/assets/pages/weekly-subscription-landing/components/hero/hero.jsx

```diff
@@ -78,7 +78,7 @@ const CampaignHeader = () => (
   <ProductPagehero
     appearance="campaign"
     overheading="Guardian Weekly subscriptions"
-    heading="The Guardian's essential news magazine"
+    heading="Pause for thought with the Guardian's essential news magazine"
```

Comment:
I think it is `The Guardian` with a capital T - although I have lost track of the number of times I've seen this debated 😆

Reply:
blame garnett design updates for that!